.changeset/nasty-drinks-judge.md

```diff
@@ -0,0 +1,27 @@
+---
+'@makeswift/runtime': minor
+---
+
+This change updates the runtime to use the latest version of
+`@makeswift/controls`. As part of this update, this package is no longer
+exposing internal data types and functions associated with our controls.
+
+## BREAKING:
+
+1. Registering controls with arbitrary options will now result in a TypeScript
```

Comment:
@arvinpoddar How about this:

1. An attempt to instantiate a control with extraneous configuration parameters will now result in a TypeScript compilation error:

    ```typescript
     import { Number } from '@makeswift/runtime/controls'
    
     const num = Number({ foo: 'bar' }) // error, `foo` is not a valid `Number` param
     ```
     Prior to this version, the extraneous parameters were silently ignored.

?

Reply:
Well said, thank you